### ISSUE TITLE
Not rly an issue,but a drop situation

### ISSUE THREAD

**Issue by Crysty666** [Iskallia/Vault-public-S1#74]
I tried in my world vaults at lv 0, i found that traders are rare, idk if they become more frequent, but in comparison with iskall stream they are, i think a better way to get us some traders is by a rendom timer, and for gift bomb, i found none in vaults. i think a comand that decide how often we can get bombs, traders, boosters etc. sth like in his lives, but with no livestream.
 Idk. this is my proposal.

**Comment by Bright7185** [comment]
If you compare to antonio ashes streams then we get around the same iskall has 3k viewers on average in his stream antonio gets 300 to 400 with raids so iskall gets way more traders then antonio and it is to make you work to get traders

**Comment by Crysty666** [comment]
good point, but still get some, this thing could vary with difficulty, on easy you have 1 trader in 10-15 min , normal 1 20-25, hard 30 min+ or sth, i mean they got pogs and elytras from first stream, idk i think is a lil bit too hard, iskall said we will not have a lot of items for crystals like them 11k etc so the game so that the game is not so difficult ,but is hard bcs we dont have any traders rn, i think only way to get them it is from treasure room. developers can try to put sth in config so we can activate for us, I'm not that good at writing mods to do it my self , but to modify sth i can try. 
Again, this is my opinion, some may agree with me, others may not. keep it fun ^^
